src/forms/TimeInput.jsx

```diff
@@ -41,42 +134,139 @@ class TimeInput extends React.Component {
 			className
 		);
 
+		const fauxInputClassNames = cx(
+			'fauxInput fauxInput--time',
+			{
+				disabled,
+				error
+			}
+		);
+
+		const meridianClassNames = cx(
+			MERIDIAN_INPUT_CLASS,
+			'flush--all border--none field--reset',
+			{
+				disabled,
+				error
+			}
+		);
+
 		const labelClassNames = cx(
 			'label--field',
 			{ required }
 		);
 
 		const errorId = `${id}-error`;
 
+		const formatHours = (hours, meridian) => {
```

Comment:
this function should be defined outside of the component, not every time `render` runs